apps/dialtone-documentation/docs/design/colors/themes/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Themes
+description: Comprehensive theming system that enables consistent visual experiences across different contexts and user preferences within Dialpad products.
+no_preview: true
+---
+
+<div class="d-m32"></div>
+
+## Overview
+
+Dialtone's theming system provides a flexible foundation for creating consistent visual experiences across different contexts, user preferences, and product variations. Our theming approach leverages semantic color tokens to enable seamless switching between light, dark, and custom brand themes while maintaining accessibility and design consistency.
```

Comment:
Let's be sure we define "theme" very specifically, including paired with a visual. That is, the only themeable segment is the "shell" – top and left (at least for now).

Reply:
Let's also distinguish between `theme` and `mode`, i.e.

`mode`
* light
* dark
* high-contrast (future)

`theme`
* dp (default)
* tmo
* sunflower
* etc
* etc

Reply:
To that end... change the second and third dropdowns here...

<img width="1518" height="344" alt="Image" src="https://github.com/user-attachments/assets/473a2a34-9f09-4916-9e04-2f695a5c5360" />

...to...

<img width="1518" height="344" alt="Image" src="https://github.com/user-attachments/assets/ab183401-850b-4050-8c57-e71f709e304e" />

And log a Jira Work Item for DT Engineer to reflect this more specifically, i.e...

```html
<html 
  ...
  data-dt-theme="light"
  data-dt-brand="dp"
  ...
>
```
...changes to 
```html
<html 
  ...
  data-dt-theme="dp|tmo|sunflower|etc"
  data-dt-mode="light|dark|high-contrast(future)"
  ...
>
```
This would be a breaking change.